modules/scripts/config.py

```diff
@@ -101,6 +102,7 @@ def _get_env_dict(conf):
     build = conf['build']
     publish = conf['publish']
     extras = conf['extras']
+    social = conf['social']
```

Comment:
Ok, it seems the option in the `.yaydoc.yml` file is only used to add the directive to the auto-generated index. This setting would have no effect when using a manual index right? I think providing a directive should have been enough.